impl/src/main/java/com/okta/sdk/impl/ds/DefaultDataStore.java

```diff
@@ -278,55 +275,52 @@ private ResourceAction getPostAction(ResourceDataRequest request, Response respo
         // Most Okta endpoints do not support partial update, we can revisit in the future.
         final Map<String, Object> props = resourceConverter.convert(abstractResource, false);
 
-        FilterChain chain = new DefaultFilterChain(this.filters, new FilterChain() {
-            @Override
-            public ResourceDataResult filter(final ResourceDataRequest req) {
+        FilterChain chain = new DefaultFilterChain(this.filters, req -> {
 
-                CanonicalUri uri = req.getUri();
-                String href = uri.getAbsolutePath();
-                QueryString qs = uri.getQuery();
+            CanonicalUri uri1 = req.getUri();
```

Comment:
So, you can't reuse variable names with 2 lambda expressions in the same method?

Reply:
It should be possible, it would be scoped different, it's just not clear when they are reused

Reply:
In that case, I'd have preferred the first name to be `uri1` and the 2nd to be `uri2`. Just a suggestion. Approved anyway :)